### ISSUE TITLE
Create database backups

### ISSUE THREAD

**Issue by hobbyquaker** [hobbyquaker/node-red-contrib-zigbee#76]
#### When?
* always when setting permitJoin false?
* always after an successful deviceInterview?
* at fixed intervals?
* a combination of the above?

#### Retain?
* auto delete backups that exceed a defined range?

**Comment by bohtho** [comment]
Good idea. Where is the database stored? Would I be able to test and increasingly add devices to this node on a mac, and then install the same node on my RPi3 and bring with me the database file? Any config files also needed?

**Comment by Sineos** [comment]
Since there are reports on lost / empty databases: https://github.com/Koenkk/zigbee2mqtt/issues/3238
